packages/catalog-process/src/services/catalogService.ts

```diff
@@ -241,9 +234,9 @@ export function catalogServiceBuilder(
       authData: AuthData
     ): Promise<EService> {
       logger.info(`Retrieving EService ${eserviceId}`);
-      const eservice = await retrieveEService(eserviceId, readModelService);
+      const eService = await retrieveEService(eserviceId, readModelService);
```

Comment:
Why is `eservice` being turned into `eService` again?

Reply:
I have reverted the change to make the PR as small as possible